netjson-monitoring.lua

```diff
@@ -72,6 +72,39 @@ netjson = {
     }
 }
 
+-- collect interface addresses
+function get_addresses(name)
+    interface_data = ubus:call('network.interface', 'dump', {})
+    addresses = {}
+    interface_list = interface_data['interface']
+    for _, interface in pairs(interface_list) do
+        if interface['l3_device'] == name then
+            if interface['ipv4-address'] and next(interface['ipv4-address'])then
+                family = 'ipv4-address'
+            elseif interface['ipv6-address'] and next(interface['ipv6-address'])then
+                family = 'ipv6-address'
+            end
+            if interface[family] then
+                for _,address in pairs(interface[family]) do
+                    if address['proto'] == 'static' then
+                        proto = 'static'
+                    else
+                        -- expected values are: dhcp, dhcpv6
+                        proto = 'dhcp'
+                    table.insert(addresses, {
+                        address = address['address'],
+                        mask = address['mask'],
+                        proto = proto,
+                        family = family
```

Comment:
you must send either `ipv4` or `ipv6`, not `ipv#-address`!

Reply:
My bad